tools/data/build_rawframes.py

```diff
@@ -29,10 +30,28 @@ def extract_frame(vid_item, dev_id=0):
 
     if task == 'rgb':
         if args.use_opencv:
+            # Not like using denseflow,
+            # Use OpenCV will not make a sub directory with the video name
+            video_name = vid_path.split('.')[0].split('/')[-1]
```

Comment:
use os.path.basename or os.path.splitext to manipulate paths. avoid string operations when possible